src/Serilog.Sinks.Amazon.Kinesis/Common/HttpLogShipperBase.cs

```diff
@@ -268,40 +237,36 @@ static bool IsUnlockedAtLength(string file, long maxLen)
             catch (IOException ex)
             {
                 var errorCode = Marshal.GetHRForException(ex) & ((1 << 16) - 1);
-
-                if (errorCode == 32)
+                if (errorCode != 32 && errorCode != 33)
```

Comment:
I seem to remember that errorCode 32 was related to IO, why are we reverting the logic here?